front/src/app/todo-list/components/form-field-layout/form-field-layout.component.ts

```diff
@@ -0,0 +1,29 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, DoCheck, ElementRef, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+
+@Component({
+  selector: 'app-form-field-layout',
+  templateUrl: './form-field-layout.component.html',
+  styleUrl: './form-field-layout.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FormFieldLayoutComponent implements DoCheck {
+  @Output() taskListInput = new EventEmitter<string>();
+  @Output() taskListInputBlur = new EventEmitter();
+  @Input() taskListInputValue: string = '';
+  @ViewChild('inputLayout') inputLayout: ElementRef | undefined;

```

Comment:
A common order of properties for such component is `@Input`s, `@Outputs`, [empty line to separate public properties from the rest of the code], `@ViewChild etc